WickedEngine/wiMath.h

```diff
@@ -5,8 +5,21 @@
 #include <algorithm>
 #include <limits>
 
-#define _XM_FMA3_INTRINSICS_
```

Comment:
I want to keep these switchable only here like it was before and not use the WICKED_USE defines.